packages/config/src/handlers/application/applicationConfigHandler.ts

```diff
@@ -22,6 +22,16 @@ import { getDocumentUrl } from '@config/services/documents'
 import { COUNTRY_CONFIG_URL } from '@config/config/constants'
 import fetch from 'node-fetch'
 
+export const SystemRoleType = [
+  'FIELD_AGENT',
+  'LOCAL_REGISTRAR',
+  'LOCAL_SYSTEM_ADMIN',
+  'NATIONAL_REGISTRAR',
+  'NATIONAL_SYSTEM_ADMIN',
+  'PERFORMANCE_MANAGEMENT',
+  'REGISTRATION_AGENT'
+]
```

Comment:
Well the sys admin & performance management roles don't really have anything to do with declarations so I guess we don't need them here?